docs/hoon/twig/wut-test/bar-or.md

```diff
@@ -10,13 +10,25 @@ sort: 9
 ## Expands to
 
 ```
+:loop
 :ifno  p  
```

Comment:
I think the |- and ?~ are part of the macro code traversing the list of option twigs, and not the expanded output.